android/app/src/components/Button/index.tsx

```diff
@@ -0,0 +1,34 @@
+import {StyleSheet, Text, TouchableOpacity} from 'react-native';
+import React from 'react';
+
+interface Props {
+  toShow: string;
+  isOperation: boolean;
+  handleOnPress: (toShow: string, isOperation: boolean) => void;
+  styles: any;
+}
+
+const Button: React.FC<Props> = ({
+  toShow,
+  isOperation,
+  handleOnPress,
+  styles,
+}) => {
+  return (
+    <TouchableOpacity
+      style={styles}
+      onPress={() => handleOnPress(toShow, isOperation)}>
```

Comment:
Realiza una constante `onPress = () => handleOnPress(toShow, isOperation);`
```suggestion
      onPress={onPress}>
```